site/content/en/docs/tasks/monitor_pending_workloads/pending_workloads_on_demand.md

```diff
@@ -0,0 +1,499 @@
+---
+title: "Pending Workloads on-demand"
+date: 2023-12-05
+weight: 3
+description: >
+---
+
+This page shows you how to monitor pending workloads with VisibilityOnDemand feature.
+
+The intended audience for this page are [batch administrators](/docs/tasks#batch-administrator), and [batch users](/docs/tasks#batch-user) for [Local Queue Visibiility section](#local-queue-visibility).
+
+From version v0.6.0, Kueue provides the ability for a batch administrators to monitor
+the pipeline of pending jobs, and help users to estimate when their jobs will
+start.
+
+## Before you begin
+
+Make sure the following conditions are met:
+
+- A Kubernetes cluster is running.
+- The kubectl command-line tool has communication with your cluster.
+- [Kueue is installed](/docs/installation) in version v0.6.0 or later.
+- You have sufficient permissions according to [Setup RBAC section](#setup-rbac)
+
+### Enabling feature VisibilityOnDemand
+
+VisibilityOnDemand is an `Alpha` feature disabled by default, check the [Change the feature gates configuration](/docs/installation/#change-the-feature-gates-configuration) section of the [Installation](/docs/installation/) for details.
+
+## Monitor pending workloads on demand
+
+> _Available in Kueue v0.6.0 and later_
+
+To install a simple setup of cluster queue
+
+{{< include "examples/admin/single-clusterqueue-setup.yaml" "yaml" >}}
+
+
+run the following command:
+
+```shell
+kubectl apply -f https://kueue.sigs.k8s.io/examples/admin/single-clusterqueue-setup.yaml
+```
+
+
+Now, let's create 10 jobs
+
+{{< include "examples/jobs/sample-job.yaml" "yaml" >}}
+
+using a command: 
+
+```shell
+for i in {1..10}; do kubectl create -f https://kueue.sigs.k8s.io/examples/jobs/sample-job.yaml; done
```

Comment:
```suggestion
for i in {1..5}; do kubectl create -f https://kueue.sigs.k8s.io/examples/jobs/sample-job.yaml; done
```

I think that only 5 jobs should be enough since the below response api-server is a bit huge.

Reply:
Or, we may be able to omit the part of the below responses.